.github/workflows/go-test-coverage.yml

```diff
@@ -12,7 +12,7 @@ on:
 permissions: read-all
 
 env:
-  EXPECTED_GO_VERSION: "1.20"
+  EXPECTED_GO_VERSION: "1.21"
```

Comment:
Do we need to bump this?

Reply:
Unfortunately yes, one of the libraries has a hard requirement on it.